subsys/net/lib/aws_iot/src/aws_iot.c

```diff
@@ -9,6 +9,10 @@
 #include <net/aws_fota.h>
 #endif
 
+#if defined(CONFIG_BOARD_QEMU_X86)
```

Comment:
`certificates_provision()` is conditionally compiled with `!defined(CONFIG_BSD_LIBRARY)`, and I think that would make sense here as well.